### DISCUSSION TITLE
Question: Drivers to Connect to GS400/DB2 iSeries and create ERD

### DISCUSSION THREAD

**Discussion by ioannisvasilas** [dbeaver/dbeaver#11141] [Q&A] [answered]
Hello,

I would like to use DBeaver to create the entity relationship diagram of a DB2 iSeries database.  

I have read that in order to connect DBeaver to DB2, drivers are needed. Can you provide a list of the drivers? Does the application on top on DB2 plays any role?

Thank you in advance.

**Comment by kseniiaguzeeva** [comment, marked as answer]
DBeaver already has DB2 drivers. Select which one you need in [New database connection](https://github.com/dbeaver/dbeaver/wiki/Create-Connection).
If you want to add some specific driver, [this article](https://github.com/dbeaver/dbeaver/wiki/Database-drivers) should help you.

**Comment by ioannisvasilas** [comment]
Thank you Kseniuia! Much appreciated!

**Comment by kseniiaguzeeva** [comment]
you are welcome)
